### PR TITLE
 stub storage differently so tests work in Firefox and IE

### PR DESCRIPTION
**Pull request by Feiyang1**

### CODE REVIEW

File: packages/performance/src/services/remote_config_service.test.ts [hunk 1]
```diff
@@ -69,7 +70,12 @@ describe('Performance Monitoring > remote_config_service', () => {
 
   beforeEach(() => {
     fetchStub = stub(self, 'fetch');
-    storageGetItemStub = stub(self.localStorage, 'getItem');
+    // we need to stub the entire localStorage, because storage can't be stubbed in Firefox and IE.
+    // stubbing on self(window) seems to only work the first time (at least in Firefox), the subsequent
+    // tests will have the same stub. stub.reset() in afterEach doesn't help either. As a result, we stub on ApiInstance
```
Comment (alikn) [inline packages/performance/src/services/remote_config_service.test.ts]: Line length. Also, missing period at the end.

File: packages/performance/src/services/remote_config_service.test.ts [hunk 2]
```diff
@@ -88,15 +94,18 @@ describe('Performance Monitoring > remote_config_service', () => {
...
-        storageGetItemFakeFactory(
-          EXPIRY_LOCAL_STORAGE_VALUE,
-          STRINGIFIED_CONFIG
+      const getItemStub = stub();
+      storageStub.value({
+        getItem: getItemStub.callsFake(
+          storageGetItemFakeFactory(
+            EXPIRY_LOCAL_STORAGE_VALUE,
+            STRINGIFIED_CONFIG
+          )
```
Comment (alikn) [inline packages/performance/src/services/remote_config_service.test.ts]: can we put the stubbing part in a function which returns the stub? Something like the pattern discussed here: go/sifers.

Comment (Feiyang1) [reply packages/performance/src/services/remote_config_service.test.ts]: Done. PTAL.

File: packages/performance/src/services/remote_config_service.test.ts (hunk 1 above)
Comment (Feiyang1) [reply packages/performance/src/services/remote_config_service.test.ts]: done.

**Pull request merged**